web/apps/apps_test.go

```diff
@@ -428,6 +428,7 @@ func TestApps(t *testing.T) {
 		attrs.Value("Cookie").String().Contains("HttpOnly")
 		attrs.Value("Token").String().NotEmpty()
 		attrs.ValueEqual("Flags", "{}")
+		attrs.Value("Warnings")
```

Comment:
hu, what are you testing here?

Reply:
I thought `attrs.Value("Warnings").String().NotEmpty()` would throw since there are no warnings but it doesn't and probably does the actual presence check.